diff --git a/src/lib/default.nix b/src/lib/default.nix
index 4772b6d..b43ca70 100644
--- a/src/lib/default.nix
+++ b/src/lib/default.nix
@@ -1,8 +1,5 @@
-styx:
+styx: nixpkgs:
 let
-  # For runCommand and writeText
-  nixpkgs = import <nixpkgs> {};
-
   # nixpkgs lib
   base = nixpkgs.lib // builtins;
   pkgs = { inherit styx; inherit (nixpkgs) runCommand writeText; };
diff --git a/themes/default.nix b/themes/default.nix
index 5d2cce2..7d5bdb4 100644
--- a/themes/default.nix
+++ b/themes/default.nix
@@ -1,6 +1,5 @@
-let
-  pkgs = import <nixpkgs> {};
-in
-  pkgs.lib.mapAttrs (n: v:
-    pkgs.callPackage (pkgs.fetchFromGitHub v) {}
-  ) (import ./versions.nix)
+{ pkgs ? import <nixpkgs> {} }:
+
+pkgs.lib.mapAttrs (n: v:
+  pkgs.callPackage (pkgs.fetchFromGitHub v) {}
+) (import ./versions.nix)
